src/main/java/net/fabricmc/tinyremapper/IMappingProvider.java

```diff
@@ -39,5 +39,20 @@ public Member(String owner, String name, String desc) {
 		public String owner;
 		public String name;
 		public String desc;
+
+		@Override
+		public int hashCode() {
+			return (31 * (31 * owner.hashCode()) + name.hashCode()) + desc.hashCode();
```

Comment:
You could use [`java.util.Objects.hash`](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#hash-java.lang.Object...-) for that